.github/workflows/release.yaml

```diff
@@ -0,0 +1,182 @@
+    steps:
+    - name: Checkout registry-operator source code
+      uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+    - name: Download yq
+      run: curl -sL https://github.com/mikefarah/yq/releases/download/v4.34.2/yq_linux_amd64 -o yq && chmod +x yq
+    - name: Cache version tags
+      env:
+        YQ_CLI: ./yq
+      run: mkdir -p .cache && bash .ci/cache_version_tags.sh
+    - name: Check if VERSION and version tags match release tag
+      run: bash check_version.sh ${{ github.ref_name }} true
+  k8s-1-26-integration-test:
+    needs: version-check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout registry-operator source code
+      uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+    - name: Setup Go environment
+      uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
+      with:
+        go-version: 1.19
+    - name: Download operator-sdk
+      run: |
+        export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+        export OS=$(uname | awk '{print tolower($0)}')
+
+        curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+    - name: Verify the downloaded operator-sdk
+      run: |
+        export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+        export OS=$(uname | awk '{print tolower($0)}')
+        gpg --keyserver keyserver.ubuntu.com --recv-keys 052996E2A20B5C7E;
+        curl -LO ${OPERATOR_SDK_DL_URL}/checksums.txt;
+        curl -LO ${OPERATOR_SDK_DL_URL}/checksums.txt.asc;
+        gpg -u "Operator SDK (release) <cncf-operator-sdk@cncf.io>" --verify checksums.txt.asc;
+        grep operator-sdk_${OS}_${ARCH} checksums.txt | sha256sum -c -
+    - name: Setup operator-sdk
+      run: |
+        export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+        export OS=$(uname | awk '{print tolower($0)}')
+        mv operator-sdk_${OS}_${ARCH} ${OPERATOR_SDK_CLI}
+        chmod +x ${OPERATOR_SDK_CLI}
+    - name: Setup Minikube
+      uses: manusa/actions-setup-minikube@3856c6fa039819f1c8e7e248b1fc5a8564e354c9 # v2.9.0
+      with:
+        minikube version: 'v1.31.2'
+        kubernetes version: 'v1.26.2'
+        driver: 'docker'
+        github token: ${{ secrets.GITHUB_TOKEN }}
+        start args: '--memory 4096 --cpus 2'
+    - name: Set manager image pull policy to Never
+      uses: mikefarah/yq@0b34c9a00de1c575a34eea05af1d956a525c4fc1 # v4.34.2
+      with:
+        cmd: yq -i '(select(documentIndex == 1) | .spec.template.spec.containers[0].imagePullPolicy) = "Never"' 'config/manager/manager.yaml'
+    - name: Run integration testing on release image
+      env:
+        REGISTRY_OPERATOR: localhost/registry-operator:${{ github.ref_name }}
+      run: bash .ci/minikube_integration.sh
+  k8s-1-25-integration-test:
```

Comment:
The only difference between the `k8s-1-26-integration-test` and `k8s-1-25-integration-test` jobs seems to be the `kubernetes version` field from `manusa/actions-setup-minikube`, right?
If so, I'd suggest using a matrix strategy to simplify this (and make it easier to maintain in the future if we want to test against other versions of Kubernetes for example). See https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
resolved in https://github.com/devfile/registry-operator/pull/56/commits/bb479e08a64c3c09d1faeb12e16eb0c5ceeae105